examples/sui-move/escrow/sources/example.move

```diff
@@ -77,14 +77,11 @@ module escrow::example {
     /// The key does not match this lock.
     const ELockKeyMismatch: u64 = 0;
 
-    /// The locked object has already been swapped
-    const EAlreadySwapped: u64 = 1;
-
     /// The `sender` and `recipient` of the two escrowed objects do not match
```

Comment:
I changed this to fix some debug errors that come out during build. (Removed unused var)